<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DesSettings</class>
 <widget class="QDialog" name="DesSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tabdesigner">
      <attribute name="title">
       <string>Designer</string>
      </attribute>
      <widget class="QGroupBox" name="groupGrid">
       <property name="geometry">
        <rect>
         <x>7</x>
         <y>5</y>
         <width>181</width>
         <height>71</height>
        </rect>
       </property>
       <property name="title">
        <string>Grid</string>
       </property>
       <widget class="QCheckBox" name="checkshowgrid">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>20</y>
          <width>151</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>show grid</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkaligngrid">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>40</y>
          <width>151</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>align by grig</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupStep">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>75</y>
         <width>181</width>
         <height>80</height>
        </rect>
       </property>
       <property name="title">
        <string>Grid step</string>
       </property>
       <widget class="QRadioButton" name="radio4point">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>15</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>4 point</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radio8point">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>35</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>8 point</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radio18point">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>53</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>18 point</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>195</x>
         <y>5</y>
         <width>151</width>
         <height>71</height>
        </rect>
       </property>
       <property name="title">
        <string>Rulers measures</string>
       </property>
       <widget class="QRadioButton" name="radiomm">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>121</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>millimetre</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioinch">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>111</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>inch</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>157</y>
         <width>341</width>
         <height>70</height>
        </rect>
       </property>
       <property name="title">
        <string>Other</string>
       </property>
       <widget class="QCheckBox" name="checkeditafterinsert">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>17</y>
          <width>181</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>edit after insert</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkshowheaders">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>181</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>show band headers</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DesSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DesSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
